Simplenote/Resources/markdown-dark-contrast.css

```diff
@@ -0,0 +1,5 @@
+
+/* Accessibility high contrast blue */
+.note-detail-markdown a {
```

Comment:
Question: I've noticed the `.note-detail-markdown a` style ends up duplicate (with the second entry being the one we want).

Do we need to use `!important` to signal this one should superceed the first one?

Reply:
I had it in before, but removed it cause it would work without it.  Back in support, the CSS guild would always shy away from !important for... reasons, but yeah I think it is probably better to have it just in case since we can't do more specific targeting